src/scaict_uwu/libs/language/language_tag_factory.py

```
@@ -0,0 +1,55 @@
+
+# Local imports
+from .language_tag import LanguageTag
+from .language_utils import LanguageUtils
+
+
+class LanguageTagFactory:
+    """
+    The LanguageTagFactory class deals with LanguageTag object creations.
+    """
+
+    _tags: dict[str, LanguageTag] = {}
```

Ruff (0.8.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)